bin/pasta_show_maintainers.py

```diff
@@ -0,0 +1,165 @@
+    parser.add_argument('query', type=str, help='Group by subsystems or maintainers')
+    parser.add_argument('--smallstat', action='store_true', help='Simple view')
+    parser.add_argument('--outfile', type=str, help='Output to a csv file')
+    parser.add_argument('--file', type=str, help='Select MAINTAINERS file')
+    parser.add_argument('--filter', type=str, help='Filter by file list')
+
+    args = parser.parse_args()
+
+    if args.file!= None:
+        all_maintainers_text = file_to_string(args.file)
+    else:
+        all_maintainers_text = file_to_string('./resources/linux/repo/MAINTAINERS')
+    
+    if args.filter!= None:
+        filenames = file_to_string(args.filter).splitlines()
+
+    all_maintainers = LinuxMaintainers(all_maintainers_text)
+
+    if args.query == 'entries':
+        title = 'Entry'
+        counts, dirs= get_entry_counts(all_maintainers)
+        if args.filter != None:
+            counts_filt, dirs_filt = get_entry_counts(all_maintainers, filenames)
+    elif args.query == 'maintainers':
+        title = 'Maintainer'
+        counts, maintainers_subs = get_maintainer_counts(all_maintainers)
+        if args.filter != None:
+            counts_filt, _ = get_maintainer_counts(all_maintainers, filenames)
+    else:
+        return -1
+
+    if args.smallstat:
+        if args.filter != None:
+            results = get_small(title, counts_filt)
+        else:
+            results = get_small(title, counts)
+    else:
+        results = list()
+        if args.filter != None and args.query == 'entries':
+            results.append([title, 'Status', 'LoC in list', 'Total LoC', 'Byte count',  'In list(%)', 'Other directories'])
+            for item in counts_filt:
+                if any(all_maintainers.subsystems[item].status):
+                    results.append([item, all_maintainers.subsystems[item].status[0].value,counts_filt[item][0], counts[item][0], counts_filt[item][1], 
+                    round((counts_filt[item][0]/counts[item][0])*100, 2), ', '.join(dirs[item].difference(dirs_filt[item]))]) 
+                else:
+                    results.append([item, '', counts_filt[item][0], counts[item][0], counts_filt[item][1], round((counts_filt[item][0]/counts[item][0])*100, 2)])
+        elif args.filter != None and args.query == 'maintainers':
+            results.append([title, 'LoC in list', 'Total LoC', 'Byte count',  'In list(%)', 'Entries'])
+            for item in counts_filt:
+                results.append([item, counts_filt[item][0], counts[item][0], counts_filt[item][1], round((counts_filt[item][0]/counts[item][0])*100, 2), ', '.join(maintainers_subs[item]) ])
+        elif args.query == 'entries':
+            results.append([title, 'Status', 'LoC', 'Byte count'])
+            for item in counts:
+                if any(all_maintainers.subsystems[item].status):
+                    results.append([item, all_maintainers.subsystems[item].status[0].value, counts[item][0], counts[item][1]]) 
```

Comment:
all_maintainers.subsystems[item].status[0].value, why is this status[0], could there actually be multiple status entries?

Reply:
There seems to be always only one but returned as a list from LinuxSubsystem